.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "editor.formatOnSave": true,
+  "eslint.validate": ["javascript", "javascriptreact"],
+  "eslint.run": "onType",
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "css.validate": false,
+  "less.validate": false,
+  "scss.validate": false
+}
```

Comment:
this is optional by user, right? should not be commited.

Reply:
Deleted!